src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
```

Comment:
- [ ] When starting app with `npm start` I am getting an error, `node_modules/redux-logger doesn't exist`. Kindly make sure that app starts without errors.